dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -121,10 +124,22 @@ public void testDashboard()
 
       Nodes li = document.query("//li");
       Assert.assertTrue(li.size() > 100);
+      ArrayList<String> coordinateList = new ArrayList<>();
+
       for (int i = 0; i < li.size(); i++) {
         String coordinates = li.get(i).getValue();
         // fails if these are not valid Maven coordinates
         new DefaultArtifact(coordinates);
+        coordinateList.add(coordinates);
+      }
+      
+      ArrayList<String> sorted = new ArrayList<>(coordinateList);
+      Comparator<String> comparator = new SortWithoutVersion();
+      Collections.sort(sorted, comparator );
+      
+      for (int i = 0; i < sorted.size(); i++) {
+        Assert.assertEquals("Coordinates are not sorted: ", sorted.get(i),
+            coordinateList.get(i));
```

Comment:
It seems like we could have much more concise assertions by using the `Truth` library

Reply:
Specific suggestions? I looked there and didn't find anything on point. I did have to replace a general equals check on the list with an iterated check on each element to find the exact elements that were sorted out of order when my test failed.

Reply:
I think you have to combine a couple things, like `containsExactly` and `inOrder`. I'm not sure what the error messaging looks like when it fails.

Consider this optional if you want to look into it.

Reply:
`Truth.assertThat(coordinateList).containsExactlyElementsIn(sorted).inOrder();`?
http://google.github.io/truth/api/latest/com/google/common/truth/Ordered.html